src/main/kotlin/car/racing/GameRecords.kt

```diff
@@ -0,0 +1,10 @@
+package car.racing
+
+class GameRecords {
+    private val _records = mutableListOf<List<Int>>()
+    val records: List<List<Int>>
+        get() = _records
+
+    fun record(cars: Cars): Boolean = _records.add(cars.getAllMoves())
```

Comment:
Boolean을 리턴할 필요가 없다면 호출부에서 사용하지 않는 경우 불필요한 리턴값은 없애는 편이 좋습니다. 
```suggestion
    fun record(cars: Cars) {
        _records.add(cars.getAllMoves())
    }
```

Reply:
네 딱히 필요성이 없는 반환값이네요. 제거할께요